src/main/java/mike/Deadline.java

```diff
@@ -0,0 +1,76 @@
+package mike;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline (subclass of task).
+ */
+public class Deadline extends Task {
+    private final String endDate;
+    private final LocalDate date;
```

Comment:
Maybe can change the naming of the fields to be more explicit? Probably something like
```java
private final String endDateString;
private final LocalDate endDate;
```